app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.nvdovin.weatherapp.presentation.main;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import com.example.nvdovin.weatherapp.R;
+import com.example.nvdovin.weatherapp.presentation.main.weather.forecast.MainRecyclerForecastFragment;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainActivity extends AppCompatActivity implements MainView {
+
+    @BindView(R.id.forecast_progress_bar)
+    ProgressBar progressBar;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        ButterKnife.bind(this);
+
+
+        MainPresenter mainPresenter = new MainPresenter(this, this);
+
+        mainPresenter.checkLastUpdateTime();
+        Bundle bundle = new Bundle();
+
+
+        //Intent testIntent = new Intent(this, HistoryActivity.class);
+        //startActivity(testIntent);
```

Comment:
I didn't pay any attention to commented code, however formatting the code before commit is very well appreciated ;)